api/proto/info/info.proto

```diff
@@ -507,6 +509,40 @@ message ZInfoNetworkInstance {
   repeated ErrorInfo networkErr = 40;
 }
 
+message UsageInfo {
+  google.protobuf.Timestamp createTime = 1;
+  uint32 refCount = 2;
+  google.protobuf.Timestamp lastRefcountChangeTime = 3; // Last time when refCount changed
+}
+
+message VolumeResources {
+  uint64 maxSizeBytes = 1; // From config
+  uint64 curSizeBytes = 2; // Current disk usage
+}
```

Comment:
This may be "aspirational" for quite some time -- but no reason not to include it (IOW I doubt we'll be able to provide correct #s for these in all possible cases).

Reply:
Agreed. And is it just the size of the volume (and snapshot) or does it also include the objects in CAS which the volume is holding a reference on?
I think not, but we need to expose more of that as we expose more of the contentTree/CAS use of storage.